whelk-core/src/main/groovy/whelk/Whelk.groovy

```diff
@@ -392,12 +392,8 @@ class Whelk implements Storage {
                 .collect { it.first }
     }
 
-    boolean isImpliedBy(String broaderIri, String narrowerIri) {
-        return relations.isImpliedBy(broaderIri, narrowerIri)
-    }
-
-    Set<String> findInverseBroaderRelations(String iri) {
-        return relations.findInverseBroaderRelations(iri)
+    Relations relations() {
+        return relations
```

Comment:
getRelations() ?

Reply:
For me `relations()` reads better, but I changed it to be consistent.

Reply:
I see your POV, I just think in "bean property" terms here (and will use it as `whelk.relations` in groovy...). We don't have much of a "fluent" style, so yes, consistent too.